packages/core/apps/core-apps-server-internal/src/core_app.ts

```diff
@@ -238,7 +238,7 @@ export class CoreAppsService {
       // Poll for updates
       combineLatest([savedObjectsClient$, timer(0, 10_000)])
         .pipe(
-          concatMap(async ([soClient]) => {
+          switchMap(async ([soClient]) => {
```

Comment:
changed to switchMap to skip any situation where piled-up events run it too many times in a row.